conf/messages

```diff
@@ -439,6 +439,12 @@ chargeType.vatPADefaultInterestDescription = charged on the protective assessmen
 chargeType.vatErrorCorrectionNoticeDefaultInterestTitle = Error correction default interest
 chargeType.vatErrorCorrectionNoticeDefaultInterestDescription = charged on assessed amount
 
+chargeType.vatErrorCorrectionLPITitle = Interest on error correction
+chargeType.vatErrorCorrection1stLPPTitle = Late payment penalty
```

Comment:
This doesn't match content on Google doc: "Penalty for late payment of error correction"